Makefile

```diff
@@ -13,15 +13,29 @@ submodules:
 secrets:
 	-rm -rf vendor/native-secrets
 	-git clone https://github.com/kickstarter/native-secrets vendor/native-secrets
-	if [ ! -d vendor/native-secrets ]; \
```

Comment:
i essentially reverted `secrets:` to what it was before adding this `if` statement; the if statement only copied over examples but not the actual secret files for when `vendor/native-secrets` exists